pkg/util/strings.go

```diff
@@ -25,12 +25,12 @@ func GetSingleRune(s string) (rune, error) {
 	if s == "" {
 		return 0, nil
 	}
-	r, sz := utf8.DecodeRuneInString(s)
+	r, _ := utf8.DecodeRuneInString(s)
 	if r == utf8.RuneError {
 		return 0, errors.Errorf("invalid character: %s", s)
 	}
-	if sz != len(s) {
-		return r, errors.New("must be only one character")
+	if l := len(s); 1 != l {
```

Comment:
I think it was correct before. Add a descriptive comment if this is indeed intended. The length of a string is the number of bytes, not the number of runes. See https://play.golang.org/p/SE0ybC7jWoN